newsfragments/4879.fixed.md

```diff
@@ -0,0 +1 @@
+ * fixed spurious `test_double` failures.
```

Comment:
For what it's worth, when editing the CHANGELOG later I would probably just omit this line as it's not really user facing.